readme.md

```diff
@@ -26,5 +26,13 @@ Any contributions you make are **greatly appreciated**.
 4. Push to the Branch (`git push origin feature/AmazingFeature`)
 5. Open a Pull Request
 
+## Building
```

Comment:
this might need to go to the `CONTRIBUTING.md` file later though

Reply:
yeah but not really, for now all building guides are small enough to fit in the readme imo